class.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class BookList {
+  constructor() {
+    this.books = [];
+    this.addBook();
+    this.loadBooks();
+  }
+
+  addBook() {
+    const addBtn = document.getElementById('add-btn');
+    addBtn.addEventListener('click', () => {
+      const storedTitle = localStorage.getItem('title');
+      const storedAuthor = localStorage.getItem('author');
+  
+      if (storedTitle && storedAuthor) {
+        document.getElementById('title').value = storedTitle;
+        document.getElementById('author').value = storedAuthor;
+      }
+  
+      const titleInput = document.getElementById('title').value;
+      const authorInput = document.getElementById('author').value;
+  
+      const book = new Book(titleInput, authorInput);
+      this.books.push(book);
+  
+      localStorage.setItem('title', titleInput);
+      localStorage.setItem('author', authorInput);
+      localStorage.setItem('books', JSON.stringify(this.books));
+  
+      this.displayBook(book);
+    });
```

Comment:
- [ ] Great effort working on your awesome book project, but as part of the requirement of this milestone user should be able to add and remove books but currently I find it difficult to add a books which makes removing of books impossible.
Kindly work on it and be sure this functionality is implemented in your project 
Thank you.